docs/src/content/docs/ops/backup-and-restore/stale-log.adoc

```diff
@@ -0,0 +1,71 @@
+---
+title: Scenario - Stale Log & Intact Storage
+---
+
+If the log contains transaction entries but the most recent offset is **lower than the last indexed transaction**, XTDB will refuse to start. This typically indicates the log has been truncated, rotated, or replaced unexpectedly, resulting in **missing historical entries** that the system relies on to maintain consistency.
+
+You may encounter an error such as:
+
+> "*Node failed to start due to an invalid transaction log state (epoch=0, offset=200) that does not correspond with the latest indexed transaction (epoch=0 and offset=289).*"
+
+
+== What Is Preserved
+
+If the storage backend remains intact, then all previously compacted and indexed data is still available.
+
+* Historical data in all compacted Arrow files (Level-0 and higher)
+* The full queryable state of the database, up to the last successfully indexed offset
+
+== What Is Lost
+
+Any recently submitted transactions _after_ the last indexed offset that were truncated or lost from the log:
+
+* These are **unrecoverable** unless separately backed up or tracked externally.
+
+== Recovery Strategy: Using Epochs
```

Comment:
Rather than including details about epochs within the individual scenarios, I'd include this as reference documentation of the log component? Good to have a guide for the scenarios, granted - but it seems a little inside-out to _primarily_ document epochs here.